src/scss/3_components/navigation/_main-nav.scss

```diff
@@ -336,7 +342,7 @@ $main-menu-span: container !default;
 
   .hamburger {
     position: absolute;
-    top: -5.7rem;
+    top: -5.2rem;
```

Comment:
This will need changed back to -5.7rem so that it appears correctly with the new search bar on smaller screens